src/libraries/Microsoft.PowerFx.Core/Types/FunctionCategories.cs

```diff
@@ -21,5 +21,6 @@ internal enum FunctionCategories : uint
         Color = 0x80,
         REST = 0x100,
         Component = 0x200,
+        CustomObject = 0x400,
```

Comment:
this may have an impact on canvas, not sure if we want to be adding a new category for this, or just wait until we've built out library support